src/t_stream.c

```diff
@@ -3495,7 +3494,7 @@ void xautoclaimCommand(client *c) {
     preventCommandPropagation(c);
 }
 
-/* XDEL <key> [<ID1> <ID2> ... <IDN>]
+/* XDEL <key> <ID1> [<ID2> ... <IDN>]
```

Comment:
and here.

Reply:
Address all comments: remove all unnecessary changes.